lib/manager.js

```diff
@@ -0,0 +1,71 @@
+const path = require('path')
+const os = require('os')
+var Spinner = require('cli-spinner').Spinner
+const { Worker } = require('./worker_threads')
+
+const spinner = new Spinner('running... %s')
+spinner.setSpinnerString('|/-\\')
+
+const numWorkers = Math.max(Math.floor(os.cpus().length * 0.75), 1)
+
+function initWorkers (opts, cb) {
+  const workers = []
+  const results = []
+
+  function startAll () {
+    spinner.start()
```

Comment:
Please check that the spinner does not scramble output if using a non-TTY, e.g. redirect the output to a file.

Reply:
Just checked, it seems to be doing the same as it does on `master`, it's printing like this

![image](https://user-images.githubusercontent.com/2510597/97297375-f5444900-1877-11eb-867d-25156aa40ac5.png)

Reply:
let's disable all spinners/progress bars for non-tty.

Reply:
progressbar is already disabled for non-tty. Just did the same for spinner.
However, the above output (`[90m` `[39m`) isn't due to that.